src/mito2/src/lib.rs

```diff
@@ -170,3 +170,6 @@ mod worker;
 /// The [RegionManifestManager](crate::manifest::manager::RegionManifestManager) manages metadata of the engine.
 ///
 mod docs {}
+
+#[allow(dead_code)]
```

Comment:
Add some comments on why you added it.

Reply:
As for #[allow(dead_code)], we will remove all `allow(dead_code)` after finish refactoring mito.